package.json

```diff
@@ -16,16 +16,17 @@
   "bugs": {
     "url": "https://github.com/vaadin/vaadin-icons/issues"
   },
+  "homepage": "https://vaadin.com/icons",
   "files": [
     "vaadin-icons.js"
   ],
   "scripts": {
-    "start": "polymer serve --port 3000"
+    "prestart": "polymer analyze vaadin-* > analysis.json",
+    "start": "polymer serve --port 3000 --open"
   },
-  "homepage": "https://vaadin.com/icons",
   "devDependencies": {
+    "@vaadin/vaadin-component-dev-dependencies": "^1.0.0",
```

Comment:
After  vaadin/vaadin-element-skeleton#199 is fixed, we will be able to drop gulp from `@vaadin/vaadin-component-dev-dependencies` and only install it where it is actually needed

It makes sense to update gulp to 4.0.0 manually here, though

Reply:
I understand. So do you want any changes here right now? After gulp is removed from `vaadin-component-dev-dependencies` and when we update this here to use the newer version we would need to add gulp dependency here explicitly since it is now needed at least for the icon set generation.

Or do you want me to already add/update the gulp dependency here?